server/src/main/java/org/candlepin/model/ConsumerCurator.java

```diff
@@ -622,6 +622,34 @@ private void validateFacts(Consumer consumer) {
         }
     }
 
+    /**
+     * @param consumers consumers to create
+     * @return created consumers
+     */
+    @Transactional
+    public Set<Consumer> bulkCreate(Set<Consumer> consumers) {
```

Comment:
These methods already exist as saveAll and deleteAll in AbstractHibernateCurator

Reply:
Done.

Reply:
:+1: